### PR TITLE
enh(Resources/details):use endpoints sent by the backend resources

### PR DESCRIPTION
**Pull request by Noha-ElAbrouki**
**video for ressource details**

https://user-images.githubusercontent.com/97687698/179041854-39007e38-85b2-4d14-902b-61f18f0257e8.mp4

### CODE REVIEW

File: www/front_src/src/Resources/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+interface ReplaceWord {
+  endpoint: string;
+  newWord: string;
+}
+
+export const replaceWord = ({ newWord, endpoint }: ReplaceWord): string => {
```
Comment (Thebarda) [inline www/front_src/src/Resources/helpers.ts]:
```suggestion
export const replaceBasename = ({ newWord, endpoint }: ReplaceWord): string => {
```

File: www/front_src/src/Resources/Listing/useLoadResources/index.ts [hunk 2]
```diff
@@ -101,6 +102,10 @@ const useLoadResources = (): LoadResources => {
   const refreshIntervalRef = useRef<number>();
 
   const refreshIntervalMs = refreshInterval * 1000;
+  const resourceDetailsEndPoint = replaceWord({
```
Comment (Thebarda) [inline www/front_src/src/Resources/Listing/useLoadResources/index.ts]:
```suggestion
  const resourceDetailsEndPoint = replaceBasename({
```

File: www/front_src/src/Resources/Listing/useLoadResources/index.ts [hunk 3]
```diff
@@ -40,6 +40,7 @@ import {
   sendingAtom,
 } from '../listingAtoms';
 import { listResources } from '../api';
+import { replaceWord } from '../../helpers';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Listing/useLoadResources/index.ts]:
```suggestion
import { replaceBasename } from '../../helpers';
```

**Review by Thebarda — COMMENTED** [review]
I think we can remove `selectedResourceParentTypeAtom`, `selectedResourceParentTypeAtom`, `selectedResourceParentIdAtom` and `selectedResourceIdAtom`.
@adr-mo Is the resource details endpoint returned in the when retrieving the resource details ? Same question for the Metrics tab ?

**Comment by adr-mo** [comment]
When calling details endpoint yes. Détails and metrics endpoints are returned

**Comment by Noha-ElAbrouki** [comment]
> I think we can remove `selectedResourceParentTypeAtom`, `selectedResourceParentTypeAtom`, `selectedResourceParentIdAtom` and `selectedResourceIdAtom`. @adr-mo Is the resource details endpoint returned in the when retrieving the resource details ? Same question for the Metrics tab ?

they are used in selectResourceDerivedAtom which is used in metrics and details /tabs/services

**Comment by Thebarda** [comment]
> > I think we can remove `selectedResourceParentTypeAtom`, `selectedResourceParentTypeAtom`, `selectedResourceParentIdAtom` and `selectedResourceIdAtom`. @adr-mo Is the resource details endpoint returned in the when retrieving the resource details ? Same question for the Metrics tab ?
> 
> they are used in selectResourceDerivedAtom which is used in metrics and details /tabs/services

I know. Just make that change

**Comment by adr-mo** [comment]
Closing this PR to open a new one

**Pull request closed without merging**